airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/ssh/SshWrappedDestination.java

```diff
@@ -62,16 +60,8 @@ public ConnectorSpecification spec() throws Exception {
 
   @Override
   public AirbyteConnectionStatus check(final JsonNode config) throws Exception {
-    try {
-      return (endPointKey != null) ? SshTunnel.sshWrap(config, endPointKey, delegate::check)
-          : SshTunnel.sshWrap(config, hostKey, portKey, delegate::check);
-    } catch (final RuntimeException e) {
-      final String sshErrorMessage = "Could not connect with provided SSH configuration. Error: " + e.getMessage();
-      AirbyteTraceMessageUtility.emitConfigErrorTrace(e, sshErrorMessage);
-      return new AirbyteConnectionStatus()
-          .withStatus(Status.FAILED)
-          .withMessage(sshErrorMessage);
-    }
+    return (endPointKey != null) ? SshTunnel.sshWrap(config, endPointKey, delegate::check)
```

Comment:
Great job consolidating the `AirbyteConnectionStatus` into one catch block at the top level 🎉